(DOCSP-34725): Add docs release notes for w/e Dec 8 (#648)
mongodb/docs-app-services: docs-release-notes.md

# December 8, 2023

## Device Sync

- Configure Sync/Sync Settings
  - Add Flutter SDK to the list of SDKs that support Data Ingest.
  - Note that attempting to change an indexed queryable field value on device causes a compensating write error. Link to the compensating write error documentation for more information.

> - Handle Errors/Sync Errors: Add attempting to change an indexed queryable field value to the list of conditions that can trigger a compensating write error.

Reviewer: ```suggestion
- Handle Errors/Sync Errors: Add "updating an existing object's indexed queryable value" to the list of conditions that can trigger a compensating write error.
```
Possible reword for clarity

## Edge Server

> - Connect to Edge Server: Move the "connect" section from the "Use MongoDB APIs with Edge Server" page. Add details about the different ways you can connect - via Device SDK or the MongoDB connection URI.

Reviewer: ```suggestion
- Connect to Edge Server: Move the "Connect" section from the "Use MongoDB APIs with Edge Server" page. Add details about the different ways you can connect - via Device SDK or the MongoDB connection URI.
```

> - Use MongoDB APIs with Edge Server: Remove the "connect" section. Link to its new location on the "Connect to Edge Server" page.

Reviewer: ```suggestion
- Use MongoDB APIs with Edge Server: Remove the "Connect" section. Link to its new location on the "Connect to Edge Server" page.
```

## Data API

- Data API Endpoints: Include that HTTP/1.1 is a requirement when making requests.
- Custom HTTPS Endpoints: Include that HTTP/1.1 is a requirement when making requests.

## Authenticate & Manage Users

- Authentication Providers/Custom JWT: Fix typo in diagram.

## Reference

- App Services Admin API
  - Schemas

>     - List Schemas: Remove schema object from documentation to match the API implementation.

Reviewer: ```suggestion
    - List Schemas: Remove `schema` object from documentation to match the API implementation.
```
Not sure why the whole thing is blue.

    - Create a Schema: Add details about JSON escaping to avoid errors when creating a schema.
  - Deployment/Create a Deployment Draft: Correct the 200 response code to 201 to match API implementation.

>   - Applications/Create a New App and Get an App: Add device-sync to the product types for the GET and POST endpoints.

Reviewer: ```suggestion
  - Applications/Create a New App and Get an App: Add `device-sync` to the product types for the GET and POST endpoints.
```
